resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -59,10 +59,10 @@ templates:
 tar_excludes:
   - .gitignore
 build_pe: {{{is-pe-build}}}
-apt_repo_name: {{{repo-name}}}
-apt_nonfinal_repo_name: {{{nonfinal-repo-name}}}
-yum_repo_name: {{{repo-name}}}
-yum_nonfinal_repo_name: {{{nonfinal-repo-name}}}
+repo_name: {{{repo-name}}}
+repo_link_target: 'puppet'
+nonfinal_repo_name: {{{nonfinal-repo-name}}}
+nonfinal_repo_link_target: 'puppet-nightly'
```

Comment:
I originally wasn't going to do this, but I'm thinking it's going to be a lot easier if we hard code these link targets in. Thoughts, @underscorgan . I know this is a public repo, and theoretically the public could be using it... but... also, this would be awesome to have